service/build.gradle

```diff
@@ -40,10 +40,10 @@ dependencies {
 	implementation group: 'com.azure.resourcemanager', name: 'azure-resourcemanager-costmanagement', version: "1.0.0-beta.7"
 	implementation group: 'com.azure.resourcemanager', name: 'azure-resourcemanager-containerservice', version: '2.48.0'
 
-	implementation group: 'bio.terra', name: 'terra-cloud-resource-lib', version: "1.2.7-SNAPSHOT"
+	implementation group: 'bio.terra', name: 'terra-cloud-resource-lib', version: "1.2.34-SNAPSHOT"
 
 	// Sam
-	implementation group: "org.broadinstitute.dsde.workbench", name: "sam-client_2.13", version: "v0.0.269"
+	implementation group: "org.broadinstitute.dsde.workbench", name: "sam-client_2.13", version: "v0.0.371"
```

Comment:
this was not strictly necessary for this PR, but I figured I'd do it while I was here.